docs/HISTORY.txt

```diff
@@ -4,8 +4,8 @@ Changelog
 1.2 (unreleased)
 ----------------
 
+- Fix migrating users whose username is the beginning of another username [elioschmutz]
 - Implement group members migration [elioschmutz]
```

Comment:
@elioschmutz Can you add newlines between changelog entries? Should be consistent with the previous ones.